iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -79,8 +79,8 @@ public enum ErrorCode
         /// request or user input.
         /// </para>
         /// </summary>
-        //[Obsolete("This error does not appear to be returned by the service.")]
-        //[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Was this mistakenly commented out?

Reply:
No, I was still checking how we can deprecate this. The problem is we have some code that refers to it, so that could would need to be updated (aka removed) if we choose to obsolete it.

Reply:
I see, so I am going to leave it commented out. Are you tracking this error code or have we added a note in the excel doc to track this?

Reply:
@drwill-ms I've re-commented out the code. Let me know if you have any other comments.